src/commonmark/en/content/developer/web-api/new-tracker.md

```diff
@@ -503,24 +503,836 @@ A table with a full reference of error codes, messages and description:
+
+To specify program enrollment dates as part of the query:
+
+    GET /api/tracker/trackedEntities?filter=zHXD5Ve1Efw:EQ:A
+        &orgUnit=O6uvpzGd5pu&program=ur1Edk5Oe2n
+        &enrollmentEnrolledAfter=2013-01-01
+        &enrollmentEnrolledBefore=2013-09-01
+
+To constrain the response to instances of a specific tracked entity you
+can include a tracked entity query parameter:
+
+    GET /api/tracker/trackedEntities?filter=zHXD5Ve1Efw:EQ:A
+        &orgUnit=O6uvpzGd5pu
+        &ouMode=DESCENDANTS
+        &trackedEntity=cyl5vuJ5ETQ
+
+By default the instances are returned in pages of size 50, to change
+this you can use the page and pageSize query parameters:
+
+    GET /api/tracker/trackedEntities?filter=zHXD5Ve1Efw:EQ:A
+        &orgUnit=O6uvpzGd5pu
+        &ouMode=DESCENDANTS
+        &page=2&pageSize=3
+
+You can use a range of operators for the filtering:
+
+|Operator|	Description|
+|---|---|
+|`EQ`|	Equal to|
+|`GT`|	Greater than|
+|`GE`|	Greater than or equal to|
+|`LT`|	Less than|
+|`LE`|	Less than or equal to|
+|`NE`|	Not equal to|
+|`LIKE`|	Like (free text match)|
+|`IN`|	Equal to one of multiple values separated by ";"|
+
+##### Response format
+
+The `JSON` response can look like the following.
+
+Please note that field filtering (`fields=...`) support is planned but not yet implemented.
+
+```json
+{
+  "instances": [
+    {
+      "trackedEntity": "IzHblRD2sDH",
+      "trackedEntityType": "nEenWmSyUEp",
+      "createdAt": "2014-03-26T15:40:36.669",
+      "createdAtClient": "2014-03-26T15:40:36.669",
+      "updatedAt": "2014-03-28T12:28:17.544",
+      "orgUnit": "g8upMTyEZGZ",
+      "inactive": false,
+      "deleted": false,
+      "relationships": [],
+      "attributes": [
+        {
+          "attribute": "VqEFza8wbwA",
+          "code": "MMD_PER_ADR1",
+          "displayName": "Address",
+          "createdAt": "2016-01-12T00:00:00.000",
+          "updatedAt": "2016-01-12T00:00:00.000",
+          "valueType": "TEXT",
+          "value": "1061 Marconi St"
+        },
+        {
+          "attribute": "RG7uGl4w5Jq",
+          "code": "Longitude",
+          "displayName": "Longitude",
+          "createdAt": "2016-01-12T00:00:00.000",
+          "updatedAt": "2016-01-12T00:00:00.000",
+          "valueType": "TEXT",
+          "value": "27.866613"
+        },
+        ...,
+        ...,
+      ],
+      "enrollments": [],
+      "programOwners": []
+    }
+  ],
+  "page": 1,
+  "total": 39,
+  "pageSize": 1
+}
+```
+
+#### Tracked Entities single object endpoint `GET /api/tracker/trackedEntities/{uid}`
+
+Purpose of this endpoint is to retrieve one tracked entities given its uid.
+
+##### Request syntax
+
+`GET /api/tracker/trackedEntities/{uid}?program={programUid}&fields={fields}`
+
+|request parameter|type|allowed values|description|
+|---|---|---|---|
+|`uid`|`String`|`uid`|Return the Tracked Entity Instance with specified `uid`|
+|`program`|`String`|`uid`| TODO - verify what exactly this param does |
```

Comment:
Just had a look at the code. It seems in the new exporter we are missing some logic;

The idea is that when you return a tracked entity, it will include some attributes.

The attributes included will primarily be TrackedEntityTypeAttributes, we think of these attributes as "safe" or "public" within the system; IE. if you have access to a TEI, these attributes will help you identify the.

However, we also have attributes that are TrackedEntityProgramAttributes. These are hidden behind a program. You can specify the program to include them in the payload. We need to verify you have access though.

Let's have a more detailed chat about this if you need more info. It's a bit complicated.

Reply:
Description: 
This parameter will return any attributes attached to the specified program, as long as the tracked entity have it.


Make an issue for the missing backend impl.